crates/bevy_window/src/window.rs

```diff
@@ -33,16 +34,28 @@ impl Default for WindowId {
 
 #[derive(Debug)]
 pub struct Window {
-    pub id: WindowId,
-    pub width: u32,
-    pub height: u32,
-    pub title: String,
-    pub vsync: bool,
-    pub resizable: bool,
-    pub decorations: bool,
-    pub mode: WindowMode,
+    id: WindowId,
+    width: u32,
+    height: u32,
+    title: String,
+    vsync: bool,
+    resizable: bool,
+    decorations: bool,
+    mode: WindowMode,
     #[cfg(target_arch = "wasm32")]
     pub canvas: Option<String>,
+    #[doc(hidden)]
+    pub command_queue: SmallVec<[WindowCommand; 4]>,
+}
```

Comment:
I'd prefer to just use Vec here. I like sticking to the standard library unless we really need an optimization. Windows won't be an allocation hot spot.